src/components/lib/localStorage.js

```diff
@@ -0,0 +1,19 @@
+export function loadLocally(key) {
+  try {
+    const serializedData = localStorage.getItem(key)
+    if (serializedData) {
```

Comment:
This should be sufficient
```suggestion
    serializedData && JSON.parse(serializedData)
```